src/components/BasicMap/BasicMap.js

```diff
@@ -290,18 +304,19 @@ class BasicMap extends PureComponent {
 
   render() {
     const { className, tabIndex, ariaLabel } = this.props;
+    const { node } = this.state;
     return (
       <div
         className={className}
-        ref={this.node}
+        ref={this.setNode}
```

Comment:
Shouldn't this by like:
```suggestion
        ref={node}
```

Reply:
NO because it comes from the state. so we have to make a setState to update it then trigerrs a re-render